modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/provider/PowwowServiceProviderUtil.java

```diff
@@ -138,138 +127,112 @@ public static String getJoinPowwowMeetingURL(
 	public static boolean getOptionAutoStartVideo(long powwowMeetingId)
 		throws PortalException {
 
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			powwowMeetingId);
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
 
 		return powwowServiceProvider.getOptionAutoStartVideo(powwowMeetingId);
 	}
 
 	public static String getOptionPassword(long powwowMeetingId)
 		throws PortalException {
 
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			powwowMeetingId);
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
 
 		return powwowServiceProvider.getOptionPassword(powwowMeetingId);
 	}
 
-	public static long getPowwowServerId(String providerType) {
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			providerType);
+	public static long getPowwowServerId() {
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
+		String providerType = PortletProps.get(PortletPropsKeys.POWWOW_PROVIDER_TYPES);
 
 		return powwowServiceProvider.getPowwowServerId(providerType);
```

Comment:
i'm curious how this `providerType` still exists?

Reply:
It still exists because it call PowwowServerLocalServiceUtil.getPowwowServers in the implement funtion

Reply:
i'm wondering why we cannot remove `PortletPropsKeys.POWWOW_PROVIDER_TYPE` and then have `powwowServiceProvider.getPowwowServerId()`? We can update `PowwowServerLocalServiceUtil.getPowwowServers` accordingly, cannot we?